contributions/catalog/catalog_rest_service.py

```diff
@@ -113,7 +125,8 @@ def index():
             cap_json = jsonutil.create_capability_json_from_contribution_json(result.json())
             tal_json = jsonutil.create_talent_json_from_contribution_json(result.json())
 
-    return render_template('contribute/home.html', cap_json=cap_json, tal_json=tal_json, show_sel=show_sel, user=user)
+    return render_template('contribute/home.html', gittag=gittag, gitsha=gitsha,
```

Comment:
the render_template('contribute/home.html') returns template not found error if running 
```
python contributions/catalog/catalog_rest_service.py
```